patterns.csv

```diff
@@ -1,8 +1,8 @@
 "Pattern"	"Label"	"Description"
-"[A-Z0-9]{2}-[A-Z0-9]{2}-[A-Z0-9]{2}"	"MAT"	"Matriculas portuguesas 0A-0A-0A"
+"(?<=[^\w-])(?:[A-Z]{2}-\d{2}-\d{2}|[A-Z]{2}-\d{2}-[A-Z]{2}|\d{2}-[A-Z]{2}-\d{2}|\d{2}-\d{2}-[A-Z]{2}|[A-Z]{2}\s?\d{2}\s?[A-Z]{2})(?![A-Z\d])(?=[^\w-])"	"MAT"	"Matriculas portuguesas 0A-0A-0A"
```

Comment:
What are we trying to catch or change on the current behaviour?

Reply:
The previous rule didn't catch new plates without "-" such as:
- AT 12 BR
- MN 11 QR

And catched invalid plates such as:
- BB-BB-AB
- ABC-12-DEF